src/formElementPlugins/dropdownChoice/src/ApplicationView.tsx

```diff
@@ -28,19 +28,22 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
     options,
     optionsDisplayProperty,
     hasOther,
+    multiSelect = false,
     default: defaultValue,
   } = parameters as {
     label?: string
     description?: string
     placeholder?: string
     search?: boolean
     hasOther?: boolean
+    multiSelect?: boolean
```

Comment:
This is a general typescript question - do you need to have the ? for multiSelect if you've defined a default = false?

Reply:
Good question -- no you don't. It would be fine to make `multiSelect` compulsory in this case (`mutliSelect: boolean`). The only reason I didn't is I didn't want someone casually browsing this to think there was any compulsory parameters for the plugin -- cos in the configs you don't want them to be compulsory.